src/defineEnvironment.ts

```diff
@@ -1,34 +1,29 @@
-// @flow
 import {_htmlGroupBuilders, _mathmlGroupBuilders} from "./defineFunction";
 
-import type Parser from "./Parser";
-import type {AnyParseNode} from "./parseNode";
-import type {ArgType, Mode} from "./types";
-import type {NodeType} from "./parseNode";
-import type {HtmlBuilder, MathMLBuilder} from "./defineFunction";
+import Parser from "./Parser";
+import { AnyParseNode } from "./parseNode";
+import { ArgType, Mode } from "./types";
+import { NodeType } from "./parseNode";
```

Comment:
In a future PR some of these imports could be collapsed.  For this PR though I let's leave them as is since it makes it easier to review. ^_^